docs/DeveloperGuide.md

```diff
@@ -1134,8 +1134,16 @@ testers are expected to do more *exploratory* testing.
 
 ### Saving data
 
-1. Dealing with missing/corrupted data files
+1. Dealing with missing data files 
+   1. Prerequisites: Missing `data/talentassistant.json`.
+   2. Test case: Delete `data/talentassistant.json` and relaunch the application.</br>
+   Expected: Application will be populated with sample data.
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
 
-1. _{ more test cases …​ }_
+2. Dealing with corrupted data files
```

Comment:
@LeeEnEn not sure if we should add a warning here that certain invalid data might cause unexpected behaviour - i think during testing we had some issues with this, but noted that this wasnt within the scope of our module?

Reply:
Hmm, so something similar to the warning that is in the user guide?

Reply:
mmhmm, not sure if we need that here tho :/ if you think not needed lmk, ill do the merge :)

Reply:
If we are referring to the warning in user guide then I think it might not be needed since we have already stated it. Or do you want to bring it up for discussion later? 🤔

Reply:
let me do the approval first then so we have the final version of the dg to reference! :)